sugarscape.py

```diff
@@ -331,6 +332,17 @@ def generateTribeTags(self, tribe):
         random.shuffle(tags)
         return tags
 
+    def getTribeCount(self):
```

Comment:
Rather than creating a list and then converting it to a dictionary, why not just use one data structure the whole time? Either a list or dictionary would be fine, but a dictionary might be better just because it makes the relationship between tribe number and count more explicit. You can keep the `"Tribe"` in the key if you want, but you could also just print it when needed in the `__str__`.

How does this look to you?
```python
    tribeCounts = {f"Tribe {i}": 0 for i in range(numTribes)}
    for agent in self.agents:
        tribeCounts[agent.tribe] += 1
    return tribeCounts
```

Alternatively, following my other comment on `tribeCounts`:
```python
    tribeCounts = [0 for i in range(numTribes)]
    for agent in self.agents:
        tribeCounts[agent.tribe] += 1
    return tribeCounts
```